terraform/10-video-hearing-wowza/modules/wowza/wowza_lb_alerts.tf

```diff
@@ -0,0 +1,51 @@
+data "azurerm_monitor_action_group" "dev" {
+  resource_group_name = "vh-infra-core-${env}"
+  name                = "Vh-Action-Group-dev-${var.env}"
+}
+
+data "azurerm_monitor_action_group" "devops" {
+  resource_group_name = "vh-infra-core-${env}"
+  name                = "Vh-Action-Group-devops-${var.env}"
+}
+
+locals {
+  wowzaLoadBalancers = {
+    private = {
+      scope = [azurerm_lb.wowza.id]
+      name  = "vh-wowza-lb-private-alert-"
```

Comment:
you should append ${var.env} to the name so for example

```
locals {
  wowzaLoadBalancers = {
    private = {
      scope = [azurerm_lb.wowza.id]
      name  = "vh-wowza-lb-private-alert-${var.env}"
####
```

then you can just directly reference "${each.value.name}" without appending "${var.env}" below on the alert name.

Reply:
Changes made